libqpdf/QPDF.cc

```diff
@@ -44,6 +45,10 @@ static char const* EMPTY_PDF =
     "110\n"
     "%%EOF\n";
 
+namespace {
```

Comment:
@deanscarff If it's this simple, I'd be inclined to accept this, as long as it doesn't mess up performance, but I doubt it will since QPDF objects themselves are relatively few. I am working on a performance benchmark system, so I will measure it. Please mimic the indentation style used everywhere else in the code. In other words, this should be
```
namespace
{
    std::atomic<unsigned long long> unique_id;
} // namespace
```
If you mimic the style and the code passes my benchmarks and CI, I will accept the PR as is. If the indentation style is not matched, then I will modify it before committing. In the end, the only difference is whose name is on the commit.

Thanks for doing this. I didn't know about `std::atomic` or anonymous namespaces before I saw this. I just started requiring C++11 in version 9.1.0, which makes things like this possible where they weren't before.
```

Reply:
Actually I've changed this to just be a static (the are some subtle differences but they don't affect correctness), and it's more consistent with the surrounding code.  Curious to know the result of the benchmarks (I suspect on x86 architectures it should actually be faster this way, since the atomic implementations are userspace).

Reply:
Uh, I should elaborate here: the function-static *is* safe in terms of initialization order, the previous (unnamed namespace / global static) was not (having a static QPDF would be undefined behaviour, and also initializing two dynamically-allocated QPDFs would cause a race on the initialization of the atomic).

The de-initialization order is still not defined, but I can't see a scenario where that would be a problem (some destructor would have to be creating a QPDF as a program terminates).